packages/snaps-controllers/src/test-utils/multichain.ts

```diff
@@ -0,0 +1,111 @@
+import type { PermissionConstraint } from '@metamask/permission-controller';
+import { SnapEndowments } from '@metamask/snaps-rpc-methods';
+import { SnapCaveatType } from '@metamask/snaps-utils';
+import { MOCK_SNAP_ID } from '@metamask/snaps-utils/test-utils';
+import type { CaipAccountId, CaipChainId, Json } from '@metamask/utils';
+
+export const BTC_CAIP2 =
+  'bip122:000000000019d6689c085ae165831e93' as CaipChainId;
+export const BTC_CONNECTED_ACCOUNTS = [
+  'bip122:000000000019d6689c085ae165831e93:128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6',
+] as CaipAccountId[];
+
+export const MOCK_BTC_ACCOUNTS = [
+  {
+    address: '128Lkh3S7CkDTBZ8W7BbpsN3YYizJMp8p6',
+    id: '408eb023-8678-4b53-8885-f1e50b8b5bc3',
+    metadata: {
+      importTime: 1729154128900,
+      keyring: { type: 'Snap Keyring' },
+      lastSelected: 1729154128902,
+      name: 'Bitcoin Account',
+      snap: {
+        enabled: true,
+        id: MOCK_SNAP_ID,
+        name: 'Bitcoin',
+      },
+    },
+    methods: ['btc_sendmany'],
```

Comment:
Nit: We renamed it to `sendBitcoin` now: https://github.com/MetaMask/accounts/blob/main/packages/keyring-api/src/btc/types.ts#L39

```suggestion
    methods: ['sendBitcoin'],
```